flink-runtime-web/web-dashboard/src/app/app.config.ts

```diff
@@ -29,6 +29,7 @@ export const COLOR_MAP = {
   RECONCILING: '#eb2f96',
   IN_PROGRESS: '#faad14',
   SCHEDULED: '#722ed1',
-  COMPLETED  : '#1890ff'
+  COMPLETED: '#1890ff',
+  RESTARTING: '#13c2c2'
```

Comment:
How did you chose the color?

Reply:
there is a design system here 
https://github.com/apache/flink/blob/76199dca7dfc6288319ed005f7afd22b0e57857f/flink-runtime-web/web-dashboard/src/styles/theme.less#L18
and for ant design 3.x version, the color is cyan-6 from https://github.com/NG-ZORRO/ng-zorro-antd/blob/45e84e8b99cd3cedb6face42538ae4faaf6dacf5/components/style/color/colors.less#L31